packages/components/src/Badge/_docs/Badge.stories.tsx

```diff
@@ -0,0 +1,57 @@
+import React from "react"
+import { Meta, StoryObj } from "@storybook/react"
+import { classNameOverrideArgType } from "~storybook/argTypes"
+import {
+  StickerSheet,
+  StickerSheetStory,
+} from "~storybook/components/StickerSheet"
+import { Badge } from "../index"
+
+const meta = {
+  title: "Components/Badge",
+  component: Badge,
+  argTypes: { ...classNameOverrideArgType, reversed: { control: "disabled" } },
```

Comment:
I'm guessing you only disabled it because of the stickersheets right?

```suggestion
  argTypes: { ...classNameOverrideArgType },
```